### PR TITLE
[zh] Update kubectl.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh] Update kubectl.md

### CODE REVIEW

File: content/zh/docs/reference/glossary/kubectl.md [hunk 1]
```diff
@@ -19,25 +20,27 @@ id: kubectl
...
 <!--
- A command line tool for communicating with a {{< glossary_tooltip text="Kubernetes API" term_id="kubernetes-api" >}} server.
+Command line tool for communicating with a Kubernetes cluster's
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}},
+using the Kubernetes API.
 -->
 
- kubectl 是用来和 {{< glossary_tooltip text="Kubernetes API" term_id="kubernetes-api" >}} 服务器进行通信的命令行工具。
-
+ kubectl 是使用 Kubernetes API 与 Kubernetes 集群的 {{<glossary_tooltip text="control plane" term_id="control-plane" >}} 进行通信的命令行工具。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/kubectl.md:39]:
```suggestion
 kubectl 是使用 Kubernetes API 与 Kubernetes
 集群的{{<glossary_tooltip text="控制面" term_id="control-plane" >}}进行通信的命令行工具。
```

File: content/zh/docs/reference/glossary/kubectl.md [hunk 2]
```diff
@@ -19,25 +20,27 @@ id: kubectl
...
 <!--more--> 
 
 <!--
-You can use kubectl to create, inspect, update, and delete Kubernetes objects.
+You can use `kubectl` to create, inspect, update, and delete Kubernetes objects.
 -->
 
-您可以使用 kubectl 创建、检查、更新和删除 Kubernetes 对象。
+您可以使用 `kubectl` 创建、检查、更新和删除 Kubernetes 对象。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/kubectl.md:46]:
```suggestion
你可以使用 `kubectl` 创建、检视、更新和删除 Kubernetes 对象。
```

File: content/zh/docs/reference/glossary/kubectl.md (hunk 1 above)
Comment (ydFu) [inline content/zh/docs/reference/glossary/kubectl.md:39]:
```suggestion
kubectl 是使用 Kubernetes API 与 Kubernetes 
集群的 {{<glossary_tooltip text="控制面" term_id="control-plane" >}} 进行通信的命令行工具。
```

File: content/zh/docs/reference/glossary/kubectl.md (hunk 2 above)
Comment (ydFu) [inline content/zh/docs/reference/glossary/kubectl.md:46]:
```suggestion
你可以使用 `kubectl` 创建、检视、更新和删除 Kubernetes 对象。
```

**Review by ydFu — COMMENTED** [review]
The new commit doesn't seem to be adjusted according to the code view.

**Pull request merged**